docs/_guide/use.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: guide
+title: Use a component
+slug: use
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+This page describes how to [use a LitElement component in your application](#use). It also describes how to make sure your deployed code is browser-ready by [building it for production](#build) and [loading the Web Components polyfills](#polyfills).
+
+## Use a LitElement component {#use}
+
+This is a general guide to using third-party LitElement components. Refer to a component's README or other documentation for specific details.
+
+To use a LitElement component in your code:
+
+1.  Install the component from npm.
+
+    ```
+    npm install --save some-package-name
+    ```
+
+2.  Import the component.
+
+    In a JavaScript module: 
+
+    ```js
+    import 'some-package-name';
+    ```
+
+    In an HTML page:
+
+    ```html
+    <script type="module">
+    import './path-to/some-package-name/some-component.js';
+    </script>
+    ```
+
+    Or:
+
+    ```html
+    <script type="module" src="./path-to/some-package-name/some-component.js"></script>
+    ```
+
+3.  Add the component to the page via markup:
+
+    ```html
+    <some-component></some-component>
+    ```
+
+## Build for production {#build}
```

Comment:
The previous section is talking about using a element built with LitElement, but this section sounds like it's talking about using LitElement itself.

Can it be changed to talk about the element, not LitElement? ie:

"Elements built with LitElement are published to npm as standard JavaScript modules so that they can be loaded with the native module loader available in all current major browsers.

But, LitElement and elements built with it, use small deviation from the current standard - they use so-called "bare specifiers" to import their dependencies. In order to be loaded by browsers, they need to be run through a light transform to resolve the bare specifiers.

This can be done with...."